pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1044,7 +1044,7 @@ func (t *Task) getfsGroupMapForNamespace() (map[string][]PVCWithSecurityContext,
 			return nil, err
 		}
 		podList := &corev1.PodList{}
-		err = srcClient.List(context.TODO(), &k8sclient.ListOptions{Namespace: ns}, podList)
+		err = srcClient.List(context.TODO(), podList, &k8sclient.ListOptions{Namespace: ns})
```

Comment:
if we are here, can move to the k8sclient.InNamespace(ns) helper func?

Reply:
Or at least standardize on one or the other 

cc @djwhatle @jortel

Reply:
In this particular case, the InNamespace seems like it would be fine. In several cases in this PR I've gone away from using InNamesapce as the newer version of InNamespace doesn't seem to lend itself to cascading calls like the old one did, but in this one case it would be fine.